src/utils.hpp

```diff
@@ -58,13 +73,13 @@ static inline std::string vulkan_version_string(uint32_t version) {
 
 #define CVK_VK_CHECK_INTERNAL(logfn, res, msg, ...) do {                       \
             if (res != VK_SUCCESS) {                                           \
-                logfn(msg " : %s", ##__VA_ARGS__, vulkan_error_string(res)); \
+                logfn(msg " : %s", vulkan_error_string(res), ##__VA_ARGS__); \
             }                                                                  \
         } while (0);
 
 #define CVK_VK_CHECK_INTERNAL_RET(logfn, res, ret, msg, ...) do {                       \
             if (res != VK_SUCCESS) {                                           \
-                logfn(msg " : %s", ##__VA_ARGS__, vulkan_error_string(res)); \
+                logfn(msg " : %s", vulkan_error_string(res), ##__VA_ARGS__); \
```

Comment:
Hmm, I think that's wrong. If `msg` contains a format specifier then `vulkan_error_string(res)` will be used as the first value which is not the intent. Since it seems no users are using format specifiers, I'd rather you do what you did in the previous iteration and make these macros accept only simple messages.